ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -186,6 +187,16 @@ Requirements are located here: [avd-requirements](../../README.md#Requirements)
 
 #### IP Extended Access-Lists
 
+AVD currently supports 2 different data models for extended ACLs:
+
+- Legacy (for compatibility with existing designs, to avoid breaking changes)
```

Comment:
I suggest changing to:

- The legacy `access_lists` data model, for compatibility with existing deployments
- The improved `ip_access_lists` data model, for access to more EOS features